pkgs/development/python-modules/azure-containerregistry/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27
+, azure-core
+, msrest
+, msrestazure
+}:
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "azure-containerregistry";
+  disabled = isPy27;
```

```suggestion
  disabled = pythonOlder "3.6";
```

see https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/containerregistry/azure-containerregistry/setup.py#L56